### ISSUE TITLE
[New Feature] ColorPicker component

### ISSUE THREAD

**Issue by kotAPI** [rad-ui/ui#644]
# 🚀 Feature Request

---

## **Feature Description**
Build a form based color picker component

[Research pending]

**Comment by ahuangg** [comment]
I'm interested in working on this, would you happen to know specific ideas you already have for this component? Were you considering building a static selection of colors or implementing a color wheel?

**Comment by ahuangg** [comment]
I agree with you that building a headless version of what Figma currently has would be a great idea. I will get started on building and will update you on my progress.
